src/main/java/org/sonarqube/gradle/SonarPropertyComputer.java

```diff
@@ -410,23 +411,20 @@ private String computeProjectKey() {
   }
 
   private static boolean isReportEnabled(Report report) {
-    if (GradleVersion.version("7.0").compareTo(GradleVersion.current()) <= 0) {
-      return report.getRequired().getOrElse(false);
-    } else {
-      return report.isEnabled();
-    }
+    return report.getRequired().getOrElse(false);
```

Comment:
Do you have a good understanding how is this approach working both with Gradle 7 and Gradle 8 at the same time?
The return type is different on Gradle 7 and 8 for the method `report.getRequired()`. So if this code gets compiled with Gradle 8 it should not work anymore if the user chooses to use Gradle 7.